amundsen_application/static/js/components/SearchPage/SearchBar/index.tsx

```diff
@@ -60,7 +60,10 @@ export class SearchBar extends React.Component<SearchBarProps, SearchBarState> {
   handleValueSubmit = (event: React.FormEvent<HTMLFormElement>) : void => {
     event.preventDefault();
     if (this.isFormValid()) {
-      const pathName = `/search?searchTerm=${this.state.searchTerm}&selectedTab=table&pageIndex=0`;
+      let pathName = '/';
+      if (this.state.searchTerm !== '') {
+        pathName = `/search?searchTerm=${this.state.searchTerm}&selectedTab=table&pageIndex=0`;
```

Comment:
What happens if you don't hard-code the `selectedTab` and `pageIndex`? I think the `SearchPage` should be in charge of automating those default parameters.

Reply:
the page will redirect automatically to the defaults (selectedTab=table and pageIndex=0), can change this